appsync-cognito-lambda-cdk/README.md

```diff
@@ -33,24 +33,20 @@ env: { account: "13xxxxxxxxxx", region: "us-east-2" },
 5. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the `./cognito-appsync-lambda-cdk/lib/cognito-auth-cdk-stack.ts` file:
 
    ```bash
-   cdk deploy
+   npx aws-cdk deploy
    ```
 
 ## Testing
```

Comment:
for testing one easy way to make this pattern long lived is to use the AWS CLI to create the user instead of going to the console

Reply:
Added CLI instructions. 
Yea, the user can be created using AppSync, but this sample is just showing how to query for those user details. It doesn't look like the original sample had the mutation enabled.